demo/pubsub_forwarder/pubsub_forwarder.c

```diff
@@ -0,0 +1,86 @@
+//
+// This software is supplied under the terms of the MIT License, a
+// copy of which should be located in the distribution where this
+// file was obtained (LICENSE.txt).  A copy of the license may also be
+// found online at https://opensource.org/licenses/MIT.
+//
+
+//
+// Forwarder example based on https://github.com/C-o-r-E/nng_pubsub_proxy
+//
+// This example shows how to use raw sockets to set up a forwarder or proxy for pub/sub.
+// 
+// An example setup for running this example would involve the following:
+// 
+//  - Run this example binary (in the background or a terminal, etc)
+//  - In a new terminal, run `nngcat --sub --dial "tcp://localhost:3328" --quoted`
+//  - In a second terminal, run `nngcat --sub --dial "tcp://localhost:3328" --quoted`
+//  - In a third terminal, run `for n in $(seq 0 99); do nngcat --pub --dial "tcp://localhost:3327" --data "$n"; done`
+//
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <nng/nng.h>
+#include <nng/protocol/pubsub0/sub.h>
+#include <nng/protocol/pubsub0/pub.h>
+
+#define PROXY_FRONT_URL "tcp://localhost:3327"
+#define PROXY_BACK_URL "tcp://localhost:3328"
+
+void panic_on_error(int should_panic, const char* format, ...)
+{
+    if (should_panic)
```

Comment:
If you use clang-format using the .clang-format I've supplied, you'll see that for if statements like this the { should be on the same line.